src/components/navigation/footer/config/index.ts

```diff
@@ -82,10 +82,10 @@ export const footerConfig = ({
             brand: { [Brand.AutoScout24]: true, [Brand.MotoScout24]: false },
           },
           link: {
-            de: '/de/sellers',
-            en: '/de/sellers',
-            fr: '/fr/sellers',
-            it: '/it/sellers',
+            de: '/de/sellers/search',
+            en: '/de/sellers/search',
```

Comment:
Should we have an English locale in path since the target page supports it?

Reply:
We are mixing both approaches. As far as I know we should redirect to german until we decide to support en.